lib/backend/lite/lite.go

```diff
@@ -205,6 +206,9 @@ func NewWithConfig(ctx context.Context, cfg Config) (*Backend, error) {
 	// Ensure that the path to the root directory exists.
 	err := os.MkdirAll(cfg.Path, os.ModeDir|defaultDirMode)
 	if err != nil {
+		if errors.Is(err, fs.ErrPermission) {
+			return nil, trace.Wrap(err, "Teleport does not have permission to write to: %v. Ensure that you are running as a user with appropriate permissions.", path)
```

Comment:
```suggestion
			return nil, trace.AccessDenied("Teleport does not have permission to write to %v", path)
```

What does the error look like if you do this instead?

Reply:
Here you go

```
INFO             Starting Teleport v14.0.0-dev common/teleport.go:546
ERROR: initialization failed
Teleport does not have permission to write to /var/lib/teleport/proc/sqlite.db
```

Reply:
Let's do that.

Reply:
updated @zmb3